pandas/util/testing.py

```diff
@@ -2790,3 +2790,37 @@ def setTZ(tz):
         yield
     finally:
         setTZ(orig_tz)
+
+
+def capture_stdout(f):
+    """
+    Capture stdout in a buffer so that it can be checked
+    (or suppressed) during testing.
+    """
```

Comment:
also move these near the *other* routine which maniuplates stdout (about encoding IIRC).